### PR TITLE
Updated One-click Deploy 

### PR DESCRIPTION
**Pull request by Akash6222**
Listed EasyNode and RapidNode both as one-click deploy under "Run your own node"

image

### CODE REVIEW

File: docs/build-on-linea/tooling/node-providers.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ title: Node providers
 ## Run your own node
 
 - [Set it up yourself](../run-a-node/index.md)
-- [One-click deploy with EasyNode](https://app.easy-node.xyz/)
+- [One-click deploy with RapidNode](https://rapidnode.xyz/dashboard)
```
Comment (emilyJLin95) [inline docs/build-on-linea/tooling/node-providers.md:18]: Are you guys the same as EasyNode? Assuming you're different, we'd list both right?

**Review by emilyJLin95 — CHANGES_REQUESTED** [review]

File: docs/build-on-linea/tooling/node-providers.md (hunk 1 above)
Comment (Akash6222) [reply docs/build-on-linea/tooling/node-providers.md:18]: Yes we are in the similar business but different companies. Yes you would probably list both. Thanks!

Comment (emilyJLin95) [reply docs/build-on-linea/tooling/node-providers.md:18]: Cool can you make the chnage?

Comment (Akash6222) [reply docs/build-on-linea/tooling/node-providers.md:18]: My bad, I"ll do the changes

**Review by emilyJLin95 — APPROVED** [review]
LGTM

**Pull request merged**